### ISSUE TITLE
The addon is not compatible with knobs @storybook/addon-knobs anymore

### ISSUE THREAD

**Issue by angelika94** [pocka/storybook-addon-vue-info#120]
**Describe the bug**
Hi there, to be honest, I don't know who to address this issue: info addon or knobs 🤷‍♀️ .  But there is a bug: If info is activated for a story - knobs work only once and then not updating anymore.

**To Reproduce**

1. Go to https://storybook-addon-vue-info.netlify.com/?path=/story/compatibilities-storybook-addon-knobs--panel
2. Click on the "Knobs" tab
3. Type something in the "Label" input.
4. See that the first time you write something - the data in the story will update, but then, if you will type something else - you will not see that changes are applying

**Expected behavior**
Stories should react on knobs changing each time

**Versions**
I have the same issue on my local storybook, and I have these versions there:
- `storybook-addon-vue-info`: 1.3.2
- `@storybook/vue`: 5.2.6

**Additional context**
Please don't mind me if it is a "knobs" issue, just want you to know these addons are not compatible anymore. 
btw Info addon is amazing!

**Comment by pocka** [comment]
Probably https://github.com/storybookjs/storybook/issues/8165 (Duplicate of #116).

---

@angelika94 

**TL;DR**
This is an addon-knobs' bug, and could be fixed at `@storybook/addon-knobs@5.3.0-alpha.9`.

Workarounds:
- Upgrading Storybook to v5.3.0-alpha (i don't know when the stable release comes BTW).
- Downgrading to `@storybook/addon-knobs@5.1.x`.
- Using addon-docs, which uses this addon's loader (docgen loader) under the hood.

I'm not sure whether the patch will be included in the future v5.2.x releases :confused:
The easiest way I think is upgrading to v5.3.0, however, it is still in an alpha stage.
